batch/batch/driver/main.py

```diff
@@ -204,8 +205,8 @@ async def get_check_invariants(request: web.Request, _) -> web.Response:
         check_incremental(db), check_resource_aggregation(db), return_exceptions=True
     )
     return json_response({
-        'check_incremental_error': incremental_result,
-        'check_resource_aggregation_error': resource_agg_result,
+        'check_incremental_error': str(incremental_result) if incremental_result else None,
+        'check_resource_aggregation_error': str(resource_agg_result) if resource_agg_result else None,
```

Comment:
Continuing from https://github.com/hail-is/hail/pull/14170/files#r1476851034

Exception error messages, particularly in Python, are often too limited in context to explain what is happening. In particular, they are always missing the exception class name. Instead, we should always serialize the exception and its stack trace as a string:

```python3
In [3]: try:
   ...:     raise ValueError()
   ...: except Exception as err:
   ...:     print(traceback.format_exception(err))
['Traceback (most recent call last):\n', '  File "<ipython-input-3-789ebf77c715>", line 2, in <module>\n    raise ValueError()\n', 'ValueError\n']
```

Reply:
Ah I must have had `traceback` at one point and couldn't remember how to do this. Thanks!